src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -212,7 +212,7 @@ $(function () {
         }
         // Do not use the parsed JSON but just a minified one not to lose number precision.
         // See: https://github.com/line/armeria/issues/273
-        argsText = JSON.minify(debugText.val());
+        argsText = JSON.minify(debugText.val().concat('\n'));
```

Comment:
- How about just using the `+` operator?
- Could you add an inline comment that explains why `'\n'` has to be added?